.github/workflows/deploy-hato-bot.yml

```diff
@@ -126,7 +126,7 @@ jobs:
           DOCKER_CMD="python --version 2>&1 | sed -e 's/^Python //g'"
           python_version=$(docker compose run hato-bot sh -c "${DOCKER_CMD}")
           echo "Python version:" "${python_version}"
-          echo "::set-output name=python_version::${python_version}"
+          echo "python_version=${python_version}" >> $GITHUB_OUTPUT
```

Comment:
https://github.com/dev-hato/hato-bot/actions/runs/3243547442/jobs/5318418820

```
.github/workflows/deploy-hato-bot.yml:125:9: shellcheck reported issue in this script: SC2086:info:4:44: Double quote to prevent globbing and word splitting [shellcheck]
    |
125 |         run: |
    |         ^~~~
```

おそらく他も同様。

Reply:
```suggestion
          echo "python_version=${python_version}" >> "${GITHUB_OUTPUT}"
```